src/helpers/selectors.ts

```diff
@@ -0,0 +1,19 @@
+import { ElementHandle, Page } from 'puppeteer';
+
+export const getElementByContent = (page: Page, text: string, type = '*'): Promise<ElementHandle | null> =>
+  page.waitForXPath(`//${type}[contains(text(), '${text}')]`);
```

Comment:
XPath is great for looking for elements with text inside.  You might consider switching `text()` for `.`

`page.waitForXPath(`//${type}[contains(., '${text}')]`);`

In my opinion, `text()` reads more clearly but in case the HTML structure changes, you might want the `.`.  I am curious, what do you think?

https://stackoverflow.com/a/41862373/1026017